src/packages/@ncigdc/components/SurvivalPlotWrapper.js

```diff
@@ -108,6 +108,7 @@ const Container = ({
 );
 
 const SurvivalPlotWrapper = ({
+  buttonStyle = {},
```

Comment:
just curious: what is this one here for?

Reply:
see my description ^^^

> add a default prop for buttonStyle because clinical analysis is crashing on develop branch

Reply:
btw buttonStyle(s) has a default in other places where it's used